src/canvas/Canvas.ts

```diff
@@ -512,6 +512,8 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
       target,
       subTargets: targets,
       dragSource: this._dragSource,
+      // this is an absolute pointer, the name convention is wrong.
+      // such was in 5.x
```

Comment:
What do we want to do with this? do we make consistency and we fix it?

Reply:
update: i normalized it. Do not merge if you don't agree.

Reply:
Yes
I think we should expose a protected method that build the event data so everything calls it